internal/collector/persistentvolume.go

```diff
@@ -108,6 +108,21 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_persistentvolume_capacity",
```

Comment:
if resource storage is the only capacity possible then this should be `kube_persistentvolume_capacity_bytes`